### DISCUSSION TITLE
Cosmetic change to /seen output

### DISCUSSION THREAD

**Discussion by Bobcat00** [EssentialsX/Essentials#4612] [Ideas and Feature Suggestions] [answered]
In the /seen command output, the player's previous names should be listed before the UUID because the following is just ugly:

![seen_output2](https://user-images.githubusercontent.com/3617206/140448701-1ca28408-51a9-46e3-b42e-2e4f2f9bd07b.png)

**Comment by JRoy** [comment, marked as answer]
Will be fixed in https://github.com/EssentialsX/Essentials/pull/4615
